### ISSUE TITLE
Support desktop notifications

### ISSUE THREAD

**Issue by PopeRigby** [linuxmint/webapp-manager#18]
I'm using Discord as a web app and, the desktop notifications don't work.

**Comment by gweissman86** [comment]
Same issue. Using Firefox with Google Messages for web. Would love ability to allow notifications.

**Comment by ryanfreckleton** [comment]
It already supports desktop notifications, at least for firefox.  Right click -> Page Info -> Permissions -> Send Notifications

**Comment by albjeremias** [comment]
> It already supports desktop notifications, at least for firefox. Right click -> Page Info -> Permissions -> Send Notifications

when do I right click?

**Comment by airbreather** [comment]
> > It already supports desktop notifications, at least for firefox. Right click -> Page Info -> Permissions -> Send Notifications
> 
> when do I right click?

Right-click wasn't working for me with Firefox.  You can probably use `Ctrl + I` (for "info") to bring up Page Info.  If the page is hijacking `Ctrl + I`, then you should still be able to use Alt to bring up the menu bar, and then Tools -> Page Info.

**Comment by tristanbiesecker** [comment]
Does anyone have a solution for this on Brave or Chrome? The suggestions in this thread don't seem to work for me.
